api/v3/Contribution.php

```diff
@@ -99,7 +99,6 @@ function _civicrm_api3_contribution_create_spec(&$params) {
   $params['contact_id']['api.required'] = 1;
   $params['total_amount']['api.required'] = 1;
   $params['payment_instrument_id']['api.aliases'] = array('payment_instrument');
-  $params['receive_date']['api.default'] = 'now';
```

Comment:
How does this fit? I suggest you make a separate issue for it since it seems like a fairly major change